Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM apache/airflow:2.9.3
+RUN pip install --no-cache-dir "apache-airflow==${AIRFLOW_VERSION}" pandas feedparser
```

Comment:
I have also created in my own a dockerfile for adding gcloud lib :) 
 
I manage to create a requirement file via poetry and used it in container.

```
COPY requirements.txt /
RUN pip install --no-cache-dir "apache-airflow==2.9.3" -r /requirements.txt
```

I think it is better to use requirement file and poetry as it manage the dependency better.

If you have issue with poetry in your computer with python, tell me and I will generate the requirement file for you ;)